src/components/navbar/NavBar.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import HomeEmptyNav from './HomeEmptyNav';
+import HomeNav from './HomeNav';
+import WelcomeNav from './WelcomeNav';
+import WhatsNav from './WhatsNav';
+import HomeNoChatYetNav from './HomeNoChatYetNav';
+
+class NavBar extends Component {
+  showNavByState(step) {
+    switch (step) {
+      case 1:
```

Comment:
I think you should declare these as constants to make the code readable. If you only use them when calling a function and here, not actually necessary, but would be better anyway.